src/app/frontend/replicasetlist/replicasetcard.html

```diff
@@ -19,11 +19,21 @@
     <div layout="column">
       <div flex layout="row" layout-align="space-between center"
           class="kd-replicaset-card-title-row">
-          <a ng-href="{{::ctrl.getReplicaSetDetailHref()}}" class="kd-replicaset-card-name" flex>
-            <h3 class="md-title kd-replicaset-card-title">
-              {{::ctrl.replicaSet.name}}
-            </h3>
-          </a>
+        <span layout="row" flex>
+        <a ng-href="{{::ctrl.getReplicaSetDetailHref()}}" class="kd-replicaset-card-name">
+          <span class="md-title kd-replicaset-card-title">
+            {{::ctrl.replicaSet.name}}
+          </span>
+        </a>
+        <md-icon class="material-icons error kd-replicaset-card-status-icon"
+                 ng-if="ctrl.hasErrors()">
+          error
+        </md-icon>
+        <md-icon ng-if="ctrl.isPending()" class="material-icons kd-replicaset-card-status-icon">
+          <!--TODO find the right icon!-->
+          warning
```

Comment:
Whoever knows this icon name. Please reply :)
![zrzut ekranu z 2016-02-01 14-08-44](https://cloud.githubusercontent.com/assets/2285385/12718214/4fcb4142-c8ed-11e5-8bb6-59da76620cb1.png)